x-pack/plugins/fleet/server/saved_objects/index.ts

```diff
@@ -143,11 +144,26 @@ const getSavedObjectTypes = (
         is_preconfigured: { type: 'boolean', index: false },
         ssl: { type: 'binary' },
         proxy_id: { type: 'keyword' },
+        shipper: {
+          properties: {
+            disk_queue_enabled: { type: 'boolean', index: false },
+            disk_queue_path: { type: 'keyword', index: false },
+            disk_queue_max_size: { type: 'integer', index: false },
+            disk_queue_encryption_enabled: { type: 'boolean', index: false },
+            disk_queue_compression_enabled: { type: 'boolean', index: false },
+            compression_level: { type: 'integer', index: false },
+            loadbalance: { type: 'boolean' },
```

Comment:
This is the only non-`index: false` field. Is that voluntary? (trying to see if all fields under `shipper` can not be indexed to ignore all properties at this level instead of defining everything in the mapping)

Reply:
Good point, I forgot this one, simply because I added this parameter at different times. I think that all of them can have `index: false`.